tests/general/test_nftables.py

```diff
@@ -0,0 +1,50 @@
+import json
+import errno
+from subprocess import check_output, CalledProcessError
+
+from pyroute2 import netns
+from pyroute2.nftables.main import NFTables
+from pyroute2.nftables.rule import NFTRule
+
+from utils import require_user
+
+#NFT_BIN_PATH = "/root/nft/nftables/src/nft"
+NFT_BIN_PATH = "nft"
+NS_NAME = 'pyroute2_test_nftable'
+
+
+class NFTables_test(object):
```

Comment:
I'm going to disable this test for a while since the nft command does not support `export json` anymore, and `-j list ruleset` provides a completely different output.

We have to rework these tests somehow.